streams/src/main/java/org/apache/kafka/streams/kstream/TimeWindowedSerializer.java

```diff
@@ -38,32 +48,42 @@ public TimeWindowedSerializer(final Serializer<T> inner) {
         this.inner = inner;
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"deprecation", "unchecked"})
     @Override
     public void configure(final Map<String, ?> configs, final boolean isKey) {
-        final String windowedInnerClassSerdeConfig = (String) configs.get(StreamsConfig.WINDOWED_INNER_CLASS_SERDE);
-        Serde<T> windowInnerClassSerde = null;
-        if (windowedInnerClassSerdeConfig != null) {
+        String serializerConfigFrom = WINDOWED_INNER_SERIALIZER_CLASS;
+        String windowedInnerSerializerClassConfig = (String) configs.get(WINDOWED_INNER_SERIALIZER_CLASS);
```

Comment:
variable name suggestion: serializerConfigValue

Also in other places. I think the key/value relationship between these two variables is helpful to understand this code.